lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -1,10 +1,71 @@
 <div class="team-profile">
-    <div class="discussion-module" data-course-id="<%= courseID %>" data-discussion-id="<%= discussionTopicID %>"
-            data-read-only="<%= readOnly %>"
-            data-user-create-comment="<%= !readOnly %>"
-            data-user-create-subcomment="<%= !readOnly %>">
-        <% if ( !readOnly) { %>
-            <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
+    <div class="page-content-main">
+        <div class="discussion-module" data-course-id="<%= courseID %>" data-discussion-id="<%= discussionTopicID %>"
+                data-read-only="<%= readOnly %>"
+                data-user-create-comment="<%= !readOnly %>"
+                data-user-create-subcomment="<%= !readOnly %>">
+            <% if ( !readOnly) { %>
+                <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
+            <% } %>
+        </div>
+    </div>
+
+    <div class="page-content-secondary">
+        <h4 class="team-detail-header"><%- gettext("Team Details") %></h4>
+        <% if (isMember) { %>
+            <div class="team-user-membership-status">
+                <p><%- gettext("You are a member of this team.") %></p>
+            </div>
+        <% } %>
+        <div class="team-members">
+            <span class="sr"><%- gettext("Team member profiles") %></span>
+            <div class="members-info"></div>
+        </div>
+
+        <div class="team-capacity">
+            <span class="sr"><%- gettext("Team capacity") %></span>
+            <span><%- membershipText %></span>
+        </div>
+
+        <% if (country) { %>
+            <div class="team-country">
+                <span class="sr"><%- gettext("The country that team members primarily identify with.") %></span>
+                <i class="u-field-icon icon fa fa-globe fa-fw" aria-hidden="true"></i>
+                <span>
+                    <%- gettext(country) %>
+                </span>
+            </div>
+        <% } %>
+        <% if (language) { %>
+            <div class="team-language">
+                <span class="sr"><%- gettext("The language that team members primarily use to communicate with each other.") %></span>
+                <i class="icon fa fa-comment-o fa-fw" aria-hidden="true"></i>
```

Comment:
The previous icon has class `u-field-icon`. Do we need to add that here or remove it from the previous icon?